### PR TITLE
Update site_update.md

### LINKED ISSUE #112: Issue Template Updates for Figma link
### Current Behavior
The project's current issue templates are missing an open invitation link where new contributors can join the community's Figma team and view user interface designs and other UX projects.

### Desired Situation
Each template that has a reference to Figma in its resources section should have an invite link added.

### Implementation 
```
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

**Acceptance Tests**
All references to Figma include the "open invite" link.

**[Contributor Guide](https://docs.meshery.io/project/contributing)**

### PR DESCRIPTION
**Pull request by Karmpatel960**
**Description**

This PR fixes #112 

**Notes for Reviewers**
I changed the open invite figma link and separation of figma and slack link.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit a49202e0fa53b02da1d75b2fbe0a7ea5d690373f at: https://64c43a93ce1ad70fdc7f7bdf--service-mesh-patterns.netlify.app

**Comment by Chadha93** [comment]
@thisiskaransgit ^^

**Comment by thisiskaransgit** [comment]
@Karmpatel960, thanks for this quick fix, please update other templates too with the required link.

File: .github/ISSUE_TEMPLATE/site_update.md
Comment (thisiskaransgit) [inline .github/ISSUE_TEMPLATE/site_update.md:1]: Thanks

Comment (thisiskaransgit) [reply .github/ISSUE_TEMPLATE/site_update.md:1]: @Karmpatel960, please ping me when you're done.

Comment (Karmpatel960) [reply .github/ISSUE_TEMPLATE/site_update.md:1]: Hey @thisiskaransgit, I have updated the other issue template also Please review it :)

**Review by thisiskaransgit — APPROVED** [review]
@Karmpatel960 Looks good to merge, thank you for your contribution :tada:

**Comment by Karmpatel960** [comment]
@Chadha93, @thisiskaransgit, @l5io  Please check it is ready to merge :)

**Pull request merged**